src/scripts/expletives.js

```diff
@@ -63,7 +63,8 @@ const englishKoreanRegex = /([\w]+|[\x00-\x7F\x80-\xFF\u0100-\u017F\u0180-\u024F
 
 /* Checks if message has expletives */
 function check(message) {
-	let contentArray = getContent(message).split(englishKoreanRegex);
+  if (!message.content) return
```

Comment:
This should have two more spaces to match the 4 space width indentation of the rest of the code

Reply:
Oops I see that